support-frontend/assets/helpers/subscriptionsForms/submit.ts

```diff
@@ -110,14 +114,14 @@ const getProduct = (
 	/* Paper or PaperAndDigital */
 	return {
 		productType: Paper,
-		currency: currencyId || state.common.internationalisation.currencyId,
+		currency: currencyId ?? state.common.internationalisation.currencyId,
 		billingPeriod,
 		fulfilmentOptions: fulfilmentOption,
 		productOptions: productOption,
 	};
 };
 
-const getPromoCode = (promotions: Promotion[] | null | undefined) => {
+const getPromoCode = (promotions: Promotion[] | undefined) => {
```

Comment:
Could we make the `promotions` argument optional and get rid of the `| undefined`?

```
const getPromoCode = (promotions?: Promotion[]) => { ... }
```